rackcity/views/it_instance_views.py

```diff
@@ -167,3 +174,15 @@ def instance_page_count(request):
     instance_count = ITInstance.objects.all().count()
     page_count = math.ceil(instance_count / page_size)
     return JsonResponse({"page_count": page_count})
+
+
+def is_location_full(rack_id, elevation, height):
+    rack_u = [elevation + i for i in range(height)]
```

Comment:
I think this could be named better too - maybe `new_instance_location_range` or something like that?

Reply:
That's kind of clunky I admit but I think rack_u is unclear

Reply:
I like